### PR TITLE
add Gamma distribution

### PR DESCRIPTION
**Pull request by YoshikawaMasashi**

### CODE REVIEW

**Comment by YoshikawaMasashi** [comment]
jenkins, test this please.

**Comment by toslunar** [comment]
Jenkins, test this please.

File: tests/chainer_tests/distributions_tests/test_kldivergence.py [hunk 1]
```diff
@@ -174,6 +180,17 @@ def test_dirichlet_dirichlet_gpu(self):
         dist2 = self.make_dirichlet_dist(True)
         self.check_kl(dist1, dist2)
 
+    def test_gamma_gamma_cpu(self):
```
Comment (toslunar) [inline tests/chainer_tests/distributions_tests/test_kldivergence.py]: require scipy

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

**Pull request merged**